cvat/apps/dataset_manager/formats/cvat.py

```diff
@@ -1674,8 +1674,8 @@ def _import(src_file, temp_dir, instance_data, load_data_callback=None, **kwargs
     if is_zip:
         zipfile.ZipFile(src_file).extractall(temp_dir)
 
+        detect_dataset(temp_dir, format_name="cvat", importer=_CvatImporter)
```

Comment:
I don't see the reason why we should check the uploaded file structure only when importing into a project

Reply:
The change looks reasonable to me, but since it's user-facing, you should add a changelog entry.